### DISCUSSION TITLE
Syndicate Wars references/inspirations

### DISCUSSION THREAD

**Discussion by Moburma** [swfans/syndwarsfx#168] [General]
Something fun I thought of ages ago, references or inspiration for stuff in the game. I didn't want to go for blatantly obvious general things like yes of course e.g. Blade Runner or Neuromancer were inspirations for the cyberpunk setting, but more specific references or - ahem - even rip-offs!

**Kaneda from Akira**

![](https://static1.cbrimages.com/wordpress/wp-content/uploads/2019/11/akira-drug-pill-shirt.jpg)

The leather jacket guy is quite blatantly a nod to the protagonist from Akira, same red jacket and pill graphic on the back. 

**Poison from Final Fight**

![](https://triodepunch.wordpress.com/wp-content/uploads/2011/07/ff1.png)

The female Unguided I think are very likely heavily inspired by this character, same long flame-like hair and skimpy punkish outfit. FF was massive in arcades/on consoles around this time too.

**Sabato Kuroi from Blade of the Immortal**

![](https://static.wikia.nocookie.net/manjipedia/images/e/e0/Kuroimain.jpg/revision/latest?cb=20110204215221)

This one is a bit of a stretch and actually probably not the case, but I always thought this particular character looked a lot like the zealots. The manga was published in Japan early enough for it to be an inspiration (1992/3 I believe), but I'm not sure when it came out in English - I first came across it from an ex-girlfriend around the turn of the century and it looks like the English releases were similarly around 1999 which would totally rule it out. That said Mike Man was clearly a big manga/anime fan so maybe he had come across the original Japanese releases.

**Don't Sell Your Dreams by the Pop Group**

I've always wondered if the guitar line on the zealot track was a cheeky steal/reference to this song. Check out the descending guitar at weirdly the exact same time of about 2:15 on both of these:

https://youtu.be/7PGmJ3aYhPI?t=136
https://youtu.be/TxJ8Szq12j4?list=PL09BFD67F07E406C2&t=135

Anything else anyone spotted?

**Comment by Moburma** [comment]
One thing that struck me, I wonder if the "second dark age" mentioned in the Judge Dredd advert is a [Fall reference](https://www.youtube.com/watch?v=NQEWwjMKtY0).

**Reply by mefistotelis** [reply]
There are voices stating we're entering a dark age every decade - now on reddit, previously on BBS'es.
I remember the term from Start Trek, I think it was TNG which came out in 1970s.
So was that even a reference to a single piece of art?

**Reply by Moburma** [reply]
No, I agree it's somewhat tenuous and a common concept, but I would bet they were familiar with that song.

**Comment by Moburma** [comment]
Several Robocop allusions, although they're in the game files.
Aside from literally calling the police "Robocop" in one of the level templates, the most interesting one is the constant reference to "Splatterpunks" throughout the .mis files. These are used to mark missions with a heavy unguided element. It's also the name of the punkish civilian underclass movement rebelling against evil corporation OCP in the not very good film Robocop 3. As such they seem an obvious inspiration for the unguided.

**Comment by Moburma** [comment]
Another slightly tenuous one - the domes in Amiga chopper classic Zeewolf:

![](https://i.ibb.co/j9DwWmLq/176.png)

Have the same function as domes in SWars, although here you have to blow them up to get the goodies inside. No real proof, but it came out in November 1994 just as SW was being worked on, and of course Mike Diskett was an Amiga fan so probably aware.

**Comment by bobf5** [comment]
So I know mentioning Terminator is kind of redundant considering the familiar exo-skeleton in the cryovat screen (among other things), but it's even more blatant here in the Japanese Windows 95 autorun screen. Red eyes! (EDIT: also a big ball of electricity in the middle like somethings teleporting in... from the future... like come on :P )
![i_cant_believe_its_not_terminator](https://github.com/user-attachments/assets/dec4c8d0-3744-47c3-bdb8-130214179481)

> Several Robocop allusions, although they're in the game files.

This is a reach, but I always felt the ridiculous power of the LR Rifle and the prospect of blowing up cars and gibbing people with a sniper-rifle was probably influenced by the [Cobra Assault Cannon](https://youtu.be/YaQI8g4NJoI?t=52) from RoboCop. The voice direction on the police also always reminded me a lot of Peter Weller's performance (especially the 'Lay down your weapons!' line).

I haven't played Cannon Fodder, but from what I can tell it was an integral influence to both Syndicate and Syndicate Wars (more immediately obvious in Syndicate, especially in the earliest preview [here](https://tcrf.net/Prerelease:Syndicate_(1993,_DOS))). The EuroCorp soldiers in Syndicate Wars have an oldschool G.I. look to them, a lot like the soldiers in Cannon Fodder.
![war](https://github.com/user-attachments/assets/7deed275-d419-452e-b61b-77e37cff12db)
![war_has_never_been_so_much_fun](https://github.com/user-attachments/assets/549a1f67-b700-494e-bf46-ced359860781)

This one's definitely a reach, but I'll humor myself; in Church mission 5 you persuade a 'Doctor Schweitzer'. [Star Trek](https://en.wikipedia.org/wiki/Heroes_and_Demons) or [real-life](https://en.wikipedia.org/wiki/Albert_Schweitzer) reference? You decide!
